### ISSUE TITLE
AttributeError: 'NoneType' object has no attribute 'iter_symbols'

### ISSUE THREAD

**Issue by whitehatnote** [jonatanSh/shelf#37]
~# python3 -m shelf --input nginx_update
Traceback (most recent call last):
  File "/usr/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/usr/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/usr/local/lib/python3.6/dist-packages/shelf/__main__.py", line 113, in <module>
    start_file_method=args.start_method, args=args)
  File "/usr/local/lib/python3.6/dist-packages/shelf/relocate.py", line 38, in make_shellcode
    return shellcode_handlers[arch](args=args)
  File "/usr/local/lib/python3.6/dist-packages/shelf/lib/shellcode.py", line 643, in wrapper
    return make_shellcode(args=args, shellcode_cls=shellcode_cls)
  File "/usr/local/lib/python3.6/dist-packages/shelf/lib/shellcode.py", line 634, in make_shellcode
    shellcode = shellcode_handler.get_shellcode()
  File "/usr/local/lib/python3.6/dist-packages/shelf/lib/shellcode.py", line 497, in get_shellcode
    shellcode_data = self.correct_symbols(shellcode_data)
  File "/usr/local/lib/python3.6/dist-packages/shelf/lib/shellcode.py", line 276, in correct_symbols
    shellcode_data=shellcode_data
  File "/usr/local/lib/python3.6/dist-packages/shelf/lib/shellcode.py", line 282, in section_build_relocations_table
    original_symbol_addresses = self.get_original_symbols_addresses()
  File "/usr/local/lib/python3.6/dist-packages/shelf/lib/shellcode.py", line 435, in get_original_symbols_addresses
    for sym in symtab.iter_symbols():
AttributeError: 'NoneType' object has no attribute 'iter_symbols'

**Comment by jonatanSh** [comment]
Thank you for reporting this issue it has been resolved in shelf 3.0.5.
```bash
python3 -m pip install py_shelf
```
[Release](https://github.com/jonatanSh/shelf/releases/tag/release_3.0.5)

**Comment by jonatanSh** [comment]
Thank you for reporting this issue it has been resolved in shelf 3.0.5.
```bash
python3 -m pip install py_shelf
```
[Release](https://github.com/jonatanSh/shelf/releases/tag/release_3.0.5)

**Comment by jonatanSh** [comment]
@whitehatnote i assume you are running the library against a stripped binary the library usually require binaries with symbols and it strip them from their symbols.
I'd be happy to hear if you succeeded in making a shellcode from the source binary you provided 🙂
